src/lib/expressionBuilder.ts

```diff
@@ -36,13 +36,13 @@ interface FilterExpression {
 export type FilterExpressions = FilterExpression | FilterExpression[] | FilterExpressions[]
 
 const buildExpression = (
-    exp: FilterExpressions,
```

Comment:
There seems to be a lot of noise from your linter/EOL. Can you remove these from the PR?